fence/blueprints/admin.py

```diff
@@ -495,6 +495,25 @@ def add_policy_to_user():
 
     return jsonify(res)
 
+@blueprint.route("/list_policies", methods=["GET"])
+@admin_login_required
+@debug_log
+def list_policies():
+    """
+    Return a list of all policies. Returns in JSON format
+    """
+    expand = request.args.get('expand', default = "")
+    if(expand == "True"):
+        expand = True
+    elif(expand == "False" or expand == ""):
```

Comment:
can you adjust this so if an argument is passed we want to check to make sure its "True" if its not true then we should throw a UserError. If no argument is passed then we just want to call current_app.arborist.list_policies(). passing the parameter False is unnecessary since list_policies has a default parameter